enterprise/api/v1/views/enterprise_customer_sso_configuration.py

```diff
@@ -146,6 +146,14 @@ def oauth_orchestration_complete(self, request, configuration_uuid, *args, **kwa
                 ' not been marked as submitted.'
             )
 
+        if error_msg := request.POST.get('error'):
+            LOGGER.error(
+                f'SSO configuration record {sso_configuration_record.pk} has failed to configure due to {error_msg}.'
+            )
+            sso_configuration_record.errored_at = localized_utcnow()
+            sso_configuration_record.save()
+            return Response(status=HTTP_200_OK)
```

Comment:
Why are we returning 200 success for an error state?

Reply:
Discussed, our system isn't failing here it's properly receiving the cancellation request from the orchestrator. I also don't want to blow up the orchestrator by returning a 400+ status code when our system did what it's supposed to